tests/unit/backend/corpora/api_server/test_v1_dataset.py

```diff
@@ -315,7 +316,13 @@ def test__get_dataset_assets(self):
             user_submitted=True,
             s3_uri="s3://mock-bucket/mock-key.h5ad",
         )
-        dataset = self.generate_dataset(self.session, id="test_dataset", artifacts=[artifact_0, artifact_1])
+        artifact_2 = dict(
+            filename=CorporaConstants.ORIGINAL_H5AD_ARTIFACT_FILENAME,
+            filetype=DatasetArtifactFileType.H5AD,
+            user_submitted=True,
+            s3_uri="s3://mock-bucket/mock-key.h5ad",
```

Comment:
for clarity on which artifact should be found in assertions below, we should rename this s3_uri to something else (right now its matching artifact_1)